api/src/opentrons/protocol_engine/types/deck_configuration.py

```diff
@@ -50,6 +50,7 @@ class AddressableArea:
 
     area_name: str
     area_type: AreaType
+    mating_surface_unit_vector: Optional[List[Union[Literal[1], Literal[-1]]]]
```

Comment:
Nitpick:

```suggestion
    mating_surface_unit_vector: Tuple[Literal[1, -1], Literal[1, -1], Literal[1, -1]]
```

* `Literal[A, B]` is a shorter way of writing `Union[Literal[A], Literal[B]]`
* Typing this as a tuple of length 3 instead of an arbitrary-length list is a slightly closer match to the underlying JSON schema

If you update this, you have to update a bunch of call sites to change `[]` to `()`, so feel free to not worry about it now.